server/models.py

```diff
@@ -743,6 +746,21 @@ class File(BaseModel):
     processing_error = Column(Text)
 
 
+class TaskName(str, enum.Enum):
+    DRAW_SAMPLE = "DRAW_SAMPLE"
+
+
+class BackgroundTask(BaseModel):
+    id = Column(String(200), primary_key=True)
+    task_name = Column(Enum(TaskName), nullable=False)
+    # All tasks must have election_id in the payload
```

Comment:
I wanted to require the election_id so that it would always be logged and hopefully make debugging easier, not for any correctness concern, so that's why it seemed kind of unnecessary in the data model. I tried putting a type constraint on the payload but the type system wasn't making it easy so I decided to just make it an informal constraint. But if you think it merits a formal field I'm open to being convinced